### DISCUSSION TITLE
`x tldr`: a new client for tldr pages using shawk

### DISCUSSION THREAD

**Discussion by edwinjhlee** [tldr-pages/tldr#11113] [Show and tell]
Hello, friends.

<img width="834" alt="image" src="https://github.com/tldr-pages/tldr/assets/4426319/ddf3b737-a12a-45c1-a10b-16928738eb26">

<img width="853" alt="image" src="https://github.com/tldr-pages/tldr/assets/4426319/de3074a7-46c3-4905-abb3-19e037d68b76">

I am the author of x-cmd.  [website](https://x-cmd.com). 
We implement a tldr client using posix shell and awk, which come along with x-cmd distribution as a module.

We think `x tldr` is one of the best clients for tldr-pages for three reasons:

1. Comparing to the clients using nodejs, python, or other advanced programming language, x-cmd itself is lightweight. It only use posix shell, awk and curl in your system.
2. We provide an interactive tui app. You could see the demo [here](https://www.x-cmd.com/mod/tldr). As we upgrade our termianl ui framework in the future, the userbility can even be much better.
3.  We redistribute the tldr pages for two purposes:
  • In cases where GitHub is inaccessible for users in China, we provide hosting within China.
  • We believe that the original size (2MB) may be too large for certain embedded hardware. To address this, we have divided the package based on languages. Typically, users only need to download the English package (400KB), along with their preferred language if desired.

To use the `x tldr`, all you need to do is:

1. install x-cmd: `eval "$(curl https://get.x-cmd.com)"`. Reference: [get-started](https://www.x-cmd.com/start/)
2. just type `x tldr`. 

We hope tldr-pages can take x-cmd as one of the recommandation client. If you have any questions or requirements for `x tldr`. Please let us know in this issue.

**Comment by kbdharun** [comment]
Hi, thanks for creating a new client, I will convert this issue to a discussion instead, we have a dedicated category for showcasing new clients over there.
